ROADMAP.md

```diff
@@ -8,124 +7,164 @@ Notes: This document is still in review will may be heavily modified based on st
+
+[Milestones](#️-milestones) is our best educated guess (not a hard commitment) around when we plan to ship the key features.
+Where possible projects are broken down into discrete sub-projects e.g. project "A" may contain two sub-projects: A.1 and A.2
 
-[Milestones](#Milestones) is our best educated guess (not a hard commitment) around when we plan to ship the key features.
-Where possible we've broken down a project into discrete sub-projects e.g. project "A" (⚡Handshakes at the Speed of Light) contains three sub-projects and consists of A.1, A.2, and A.3
 A project is signified as "complete" once all of it's sub-projects are shipped.
 
-The [Roadmap Appendix](#Roadmap-Appendix) section describes each project in detail and lists its sub-projects (if applicable.) Here you can find the motivation for each project and goals.
+The [Appendix](#-appendix) section describes a project's high-level motivation, goals, and lists sub-projects.
 
-We've deep-linked each section header to it's corresponding GitHub Epic. Latest information on the progress of each project there can be found in the Epics and their child issues (as we will make regular updates.)
+Each Appendix header is linked to a GitHub Epic. Latest information on progress can be found in the Epics and child issues.
 
 ### Done criteria
-Our "Definition of Done" for projects/sub-projects that involve writing new protocols or modifications to existing ones will usually consist of the following:
-- [ ] Spec is merged and classified as "Candidate Recommendation"
-- [ ] (by virtue of the above) At least one major reference implementation exists
-- [ ] A well established testing criteria (defined at the outset of the project) is met (testing via Testground or otherwise)
-- [ ] Public documentation (on docs.libp2p) exists
+The "Definition of Done" for projects/sub-projects that involve writing new protocols/ modify existing ones usually consist of the following:
+- If a specification change is required:
+    - [ ] Spec is merged and classified as "Candidate Recommendation"
+    - [ ] (by virtue of the above) At least one major reference implementation exists
+- [ ] A well established testing criteria is met (defined at the outset of the project including but not limited to testing via Testground, compatibility tests with other implementations in the Release process, etc.)
+- [ ] Public documentation (on docs.libp2p.io) exists
+
+Supporting projects (such as testing or benchmarking) may have different criteria.
+
+## Benchmarking and Testing
+As mentioned in our [vision](#vision), performance and stability are core libp2p tenets. Rigorous benchmarking and testing help us uphold them. Related projects are listed in the [libp2p/test-plans roadmap](https://github.com/libp2p/test-plans/pull/56) and the [testground/testground roadmap](https://github.com/testground/testground/blob/master/ROADMAP.md). Our major priorities in Q4’22 and Q1’23 are:
```

Comment:
Lets link to the roadmap file rather than the PR (assuming we'll be merging soon).